### PR TITLE
Update sprint-schedule.md

### PR DESCRIPTION
**Pull request by kniz-raft**
### Summary of Changes
_updates sprint-schedule.md with how we work now, changes a few links to be relative_

### CODE REVIEW

**Comment by kniz-raft** [comment]
@carltonsmith adding the dev syncs looks great - thanks!

**Review by carltonsmith — APPROVED** [review]
LGTM!

File: docs/How-We-Work/team-charter/sprint-schedule.md [hunk 1]
```diff
@@ -1,58 +1,60 @@
...
 ### Thursday Sprint Day 9: Work Day!  
 - 10:00-11:30am EST: OFA + 18F Coordination
-- 10:30-10:45am EST: Standup
+- 12:00-12:15pm EST: Standup
 - Request feedback from teammates on rolling basis 
 
-### Friday Sprint Day 10: Submit Day!
-- 10:30-10:45am EST: Standup
-- Last day to request government review for work to count in this sprint 
+### Friday Sprint Day 10: Work Day!
```
Comment (ADPennington) [inline docs/How-We-Work/team-charter/sprint-schedule.md:44]: @kniz-raft every friday, dev team has optional drop-in hours from 10:30a-12:00p (ET).

Comment (kniz-raft) [reply docs/How-We-Work/team-charter/sprint-schedule.md:45]: Updated - thanks!

File: docs/How-We-Work/team-charter/sprint-schedule.md [hunk 2]
```diff
@@ -1,58 +1,62 @@
...
 ### Thursday Sprint Day 2: Work Day!   
 - 10:00-11:30am EST: OFA + 18F Coordination
-- 10:30-10:45am EST: Standup
+- 12:00-12:15pm EST: Standup
 - Request feedback from teammates on rolling basis
 
 ### Friday Sprint Day 3: Work Day!   
-- 10:30-10:45am EST: Standup
+- 10:30-12:00pm EST: Optional Dev Drop-in Hours
+- 12:00-1:00pm EST: IPT
```
Comment (lfrohlich) [inline docs/How-We-Work/team-charter/sprint-schedule.md:19]:
```suggestion
- 12:00-1:00pm EST: Integrated Project Team (IPT) with OCIO and other stakeholders
```

Comment (lfrohlich) [inline docs/How-We-Work/team-charter/sprint-schedule.md:19]:
```suggestion
- 12:00-1:00pm EST: Integrated Project Team (IPT) with OCIO and other stakeholders
```

**Review by lfrohlich — APPROVED** [review]
👍

**Pull request merged**